test/tests/transfers/multiapp_vector_pp_transfer/parent.i

```diff
@@ -55,7 +55,7 @@
     type = TransientMultiApp
     input_files = 'sub.i'
     positions = '0.25 1.25 0 0.5 1.5 0'
-    max_procs_per_app = 1
+    min_procs_per_app = 2
```

Comment:
shouldnt it work with 1? You could just delete the line

Reply:
The purpose of forcefully requiring the sub app to be distributed over 2 procs is to make sure the bug this PR fixes doesn't show up again. Do you think I should leave it at 1 proc and add a new test? Or just not test for this?

Reply:
if you dont specify anything, the sweep tests will check this from 1 to 16 processors naturally

Reply:
achieving what you want and keeping testing for 1 proc

Reply:
If that's the case, how come this bug wasn't caught earlier due to a test failure?

Reply:
because of this line
```
max_procs_per_app = 1
```
which I suggest you remove and do not replace

Reply:
Thanks for the explanation!